scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl

```diff
@@ -78,7 +78,8 @@ workflow GvsExtractAvroFilesForHail {
 
     call Utils.IsUsingCompressedReferences {
         input:
-            project_id = project_id,
+            query_project_id = project_id,
+            dest_project_id = project_id,
```

Comment:
I did not see Hail contains dest project and I don't know the use case. So I just pass the same project

Reply:
Can we make one of the project_ids optional then? I dont imagine we will ever extract avro files into another project

Reply:
actually this is fine--let's just get it working!